packages/common/src/content/_internal/ContentUiSchemaSettings.ts

```diff
@@ -19,6 +20,60 @@ export const buildUiSchema = async (
     type: "Layout",
     elements: [],
   };
+
+  if (
+    checkPermission("hub:content:workspace:settings:schedule", _context).access
+  ) {
+    uiSchema.elements.push({
+      type: "Section",
+      labelKey: `${i18nScope}.sections.schedule.label`,
+      elements: [
+        {
+          type: "Control",
+          scope: "/properties/schedule",
+          labelKey: `${i18nScope}.sections.schedule.helperText`,
+          options: {
+            // The scope for the options is the scope of the scheduler field, not the config editor
```

Comment:
@sonofflynn89 is that the pattern we do for other sections and schemas?

Reply:
@tomwayson it's a good question, I don't know. I was under the impression that all other configuration, editor fields depend on external (configuration editor) translation strings, not hard coded strings within the field itself. @abp6318 do you know of any other configuration editor fields whose options use static translation paths like you have here?

Reply:
@tomwayson @sonofflynn89 

It seems weird, but I took this approach considering the strings will most likely get reused, and rather than repeating strings in the already cluttered JSON file for the config editor, you can store fewer, shared strings in the input locales itself.

This was the original plan back in November and December, and there was even talk of a string audit by design/platform, but I never saw anything come out of it. 

If either of you think I should change this, I will, but to avoid having to change half a dozen strings in one file when it can be done with only one, I'd like to keep it this way.

Reply:
Discussed a better way to do this with Julianne and Jordan and settled on using default labels managed in opendata-ui, OR you can pass in a label+expandedHelperText from another scope!